yat-docs/docs/partner-guides/1_crypto_wallets.md

```diff
@@ -0,0 +1,304 @@
+---
+id: crypto-wallets
+slug: /crypto-wallets
+title: Crypto Wallets
+---
+
+## Overview
+
+This guide describes the Yat integration process for the wallet partners. This guide contains a curation of flows recommended for wallet partners who want to integrate Yats to simplify the experience of sending and receiving payments.
+
+## Getting Started
+
+### Pre-integration setup
+
+The Yat team and the partner will need to finalize the following before kicking off the integration:
+
+* Partner branding assets for the Yat web views (e.g. the partner’s logo that will appear in our web app)
+* URL path component for the partner (e.g. /coolwalletapp)
+* Partner’s return URL that the Yat web app redirects to once a user completes the integration steps in our web app (Flow [1](#1-allow-my-users-to-get-a-yat) & [2](#2-connect-existing-yats))
+* Partner app's deep link scheme for the redirection of the existing Yat users to the partner app through a QR code (Flow [1](#1-allow-my-users-to-get-a-yat), [2](#2-connect-existing-yats) & [8](#8-enable-yat-users-to-connect-with-partner-wallets))
+
+Once the partner path component, the QR code deep link scheme and the redirection URL have been assigned, they can carry out the rest of the steps below.
+
+## 1. Allow my users to get a Yat
+
+### Overview
+
+This flow covers the journey of partner app users purchasing a Yat. We recommend that partnering apps enable the user to enter the journey from at least 3 points:
+
+* Push notification announcing the partnership to all users
+* Splash screens on app launch
+* Menu item within the app’s Settings or Profile section
+
+### 1A. Get a Yat (Custodial Wallets)
+
+:::note
+With a custodial wallet, the wallet provider controls your private keys. Essentially, a third party protects your funds and returns them if you want to trade or send them somewhere else. Custodial wallets are often found in web-based exchange wallets.
+:::
+
+<a href="/img/flow1a-lrg.png" target="_blank"><img src="/img/flow1a-sml.png"/></a>
+
+### 1B. Get a Yat (Non-custodial Wallets)
+
+:::note
+Non-custodial crypto wallets give users complete control of their keys and funds. They can be browser-based, software installed on mobile devices or on desktops, or cold-storage wallets like hardware devices, among others.
+:::
+
+<a href="/img/flow1b-lrg.png" target="_blank"><img src="/img/flow1b-sml.png"/></a>
+
+&nbsp;  
+As the first step of the integration for an existing partner app user, the partner app makes a call to the /process endpoint to:
```

Comment:
@CjS77 @Krakaw that's weird - because we've listed /process in the current documentation right here: https://api-docs.y.at/docs/web_based_partner_integration#2a-existing-partner-app-users

Reply:
The current documentation references the old way of integrating that we have moved on from. I'm confirming but the new entrypoint should be `/link-email` @shanimal08 will clarify.